### DISCUSSION TITLE
Support for PSP (not emulator)

### DISCUSSION THREAD

**Discussion by nicolasnobelis** [mtkennerly/ludusavi-manifest#46] [Ideas]
When a PSP is connected to a PC in USB mode, the file system is visible to Ludusavi, and the saves games paths are the same for every PSP.

Would it be the interest of someone (except myself :laughing: ) if I start contributing new entries for PSP savegames ?

**Comment by mtkennerly** [comment]
Hi! You could absolutely put that into a secondary manifest in its own repository :) Here are some other secondary manifests you can use as examples:

* https://github.com/hblamo/ludusavi-emudeck-manifest
* https://github.com/BloodShed-Oni/ludusavi-extra-manifests

Then you can add the secondary manifest URL in Ludusavi, and it will download/update automatically.

**Comment by nicolasnobelis** [comment]
Done: https://github.com/nicolasnobelis/ludusavi-psp-manifest

I will add more entries little by little.

Thanks for your awesome tool :star_struck:

**Comment by nicolasnobelis** [comment]
@mtkennerly Just another question: after adding a secondary manifest, everything works but the entries are mixed with the ones from the primary manifest.

Can the UI group the detected entries by manifest? Or maybe just sort them by manifest?

**Reply by mtkennerly** [reply]
That's not possible right now, but I can look into adding an option for it. That sounds like a good idea :+1:

**Reply by mtkennerly** [reply]
This will be coming in the next release (https://github.com/mtkennerly/ludusavi/commit/a2a06e3542f8e9c1abb88ad7ced262c2f13494ac). I did it as a filter for now:

> https://github.com/user-attachments/assets/c7a83d5b-5f51-47de-bdde-67ee1f0a424d

**Comment by nicolasnobelis** [comment]
I will close this discussion then. Thanks !

**Reply by nicolasnobelis** [reply]
Thanks a lot !

I was not sure you forgot so I was considering implementing it :laughing:
